Code/Framework/AzCore/AzCore/EBus/EBusSharedDispatchMutex.h

```diff
@@ -0,0 +1,187 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include <AzCore/EBus/EBus.h>
+#include <AzCore/std/parallel/mutex.h>
+#include <AzCore/std/parallel/shared_mutex.h>
+
+namespace AZ
+{
+    /**
+     * %EBusSharedDispatchMutex is a custom mutex type that can be used with an EBus to allow for parallel dispatch calls, but still
```

Comment:
There's a random % here

Reply:
It's not random, I _believe_ (but haven't confirmed) that it's DOxygen markup. I saw it with EBusTraits in EBus.h and mirrored the syntax. Happy to remove it though if it's wrong.

Reply:
I did a quick search and couldn't find anything.  Might have been a typo to begin with?  It's not a big deal though

Reply:
I can't find any reference to using % in doxygen either, I'll remove it.

Reply:
Removed.